packages/core-js/modules/esnext.data-view.get-float16.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 var $ = require('../internals/export');
 var uncurryThis = require('../internals/function-uncurry-this');
-var unpackIEEE754 = require('../internals/ieee754').unpack;
+
+const expMask16 = 2 ** 5 - 1, significandMask16 = 2 ** 10 - 1, minSubnormal16 = 2 ** -10 * 2 ** -14, significandDenom16 = 2 ** -10;
```

Comment:
At this moment, the polyfills code should use ES3 syntax. After fixing that, please, fix linting issues.